modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testIntegration/java/com/liferay/headless/admin/taxonomy/resource/v1_0/test/TaxonomyCategoryResourceTest.java

```diff
@@ -893,6 +905,279 @@ private void _testGetTaxonomyVocabularyTaxonomyCategoriesPageFlatten(
 			taxonomyCategory1.getId());
 	}
 
+	private void _testImportTask(
+			String createStrategy, String parameter, long parameterValue,
+			ParentTaxonomyCategory parentTaxonomyCategory,
+			ParentTaxonomyVocabulary parentTaxonomyVocabulary,
+			TaxonomyCategory taxonomyCategory)
+		throws Exception {
+
+		taxonomyCategory.setId(String.valueOf(RandomTestUtil.randomLong()));
+		taxonomyCategory.setParentTaxonomyCategory(parentTaxonomyCategory);
+		taxonomyCategory.setParentTaxonomyVocabulary(parentTaxonomyVocabulary);
+
+		String className =
```

Comment:
@AliciaGarciaGarcia, can we use TaxonomyCategory.class.getName here instead of the string?